src/Kralizek.Extensions.Configuration.AWSSecretsManager/Internal/SecretsManagerConfigurationProviderOptions.cs

```diff
@@ -1,12 +1,15 @@
 using System;
+using System.Collections.Generic;
 using Amazon.SecretsManager;
 using Amazon.SecretsManager.Model;
 
 namespace Kralizek.Extensions.Configuration.Internal
 {
     public class SecretsManagerConfigurationProviderOptions
     {
-        public Func<SecretListEntry, bool> SecretFilter { get; set; } = secret => true;
+        public bool UseDefinedArnList { get; set; } = false;
```

Comment:
This one can be removed as we can infer its value from the amount of items in `DefinedSecretArns`.

Reply:
Good idea, done.